spec/spec_helper.rb

```diff
@@ -41,3 +45,11 @@ module TestClient
 def test_okta_token
   ENV.fetch('OKTA_TEST_TOKEN', 'x' * 40)
 end
+
+def test_okta_organization
+  ENV.fetch('OKTA_TEST_ORGANIZATION', 'okta-test')
+end
+
+def test_okta_api_endpoint
```

Comment:
If this is not set, it is ok. The client handles building the API endpoint using the organization above.